iree/vm/ops.h

```diff
@@ -57,6 +57,24 @@ static inline int32_t vm_shr_i32u(int32_t operand, int8_t amount) {
   return (int32_t)(((uint32_t)operand) >> amount);
 };
 
+//===------------------------------------------------------------------===//
+// Comparison ops
+//===------------------------------------------------------------------===//
+
+static inline bool vm_cmp_eq_i32(int32_t a, int32_t b) {
+  return a == b ? 1 : 0;
+}
+static inline bool vm_cmp_ne_i32(int32_t a, int32_t b) {
+  return a != b ? 1 : 0;
+}
+static inline bool vm_cmp_lt_i32s(int32_t a, int32_t b) {
+  return a < b ? 1 : 0;
+}
+static inline bool vm_cmp_lt_i32u(uint32_t a, uint32_t b) {
+  return a < b ? 1 : 0;
```

Comment:
```suggestion
static inline bool vm_cmp_lt_i32u(int32_t lhs, int32_t rhs) {
  (((uint32_t)lhs)<((uint32_t)rhs)) ? 1 : 0;

Reply:
Please make sure that our functions always consume and return `int32_t`.

Reply:
Fixed with latest push